catalyst_voices/lib/widgets/indicators/voices_error_indicator.dart

```diff
@@ -22,37 +21,17 @@ class VoicesErrorIndicator extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      padding: const EdgeInsets.symmetric(vertical: 6, horizontal: 16),
-      decoration: BoxDecoration(
-        border: Border.all(
-          color: Theme.of(context).colors.outlineBorderVariant!,
-          width: 1,
+    return VoicesIndicator(
```

Comment:
You're sure we want to use this in all the places? This design was supposed to be implemented by VoicesErrorIndicator: https://www.figma.com/design/80aDAQgoD0v06DO2Sb76ok/Catalyst--Minimal-Valuable-Experiments-(MVE's)?node-id=6642-93604&t=AdFNu1p5NB0czRyL-4

Reply:
Yes. Niels just re-used it [here](https://www.figma.com/design/80aDAQgoD0v06DO2Sb76ok/Catalyst--Minimal-Valuable-Experiments-(MVE's)?node-id=7058-45265&m=dev)

Reply:
Okay, does it still look the same after replacing the previous implementation by VoicesIndicator? The last time i checked voices indicator looked much differently. Can you add a screenshot with error state to the issue description please?

Reply:
Alright, I got it wrong, VoicesIndicator is a new widget and I thought it looks like VoicesStatusIndicator which is very different. No issue then, thanks 👍